### ISSUE TITLE
Fishing in ice fishing hole caused crash

### ISSUE THREAD

**Issue by HubertZhang** [penguin0616/insight#56]
```
[string "../mods/workshop-2189004162/scripts/descrip..."]:98: attempt to index field 'hook' (a nil value)
LUA ERROR stack traceback:
../mods/workshop-2189004162/scripts/descriptors/oceanfishingrod.lua:98 in (field) fn (Lua) <88-111>
   player = 116399 - wathgrithr (valid:true)
   state = table: 0x600023bcfbc0
   count = 0
   interested_fish = table: 0x600023bc9e80
   interest_levels = table: 0x600023bc9e40
scripts/scheduler.lua:186 in (method) OnTick (Lua) <164-216>
   self =
      running = table: 0x6000139d29c0
      waitingfortick = table: 0x6000139d2a00
      hibernating = table: 0x6000139d2a80
      tasks = table: 0x6000139d2980
      waking = table: 0x6000238ce900
      attime = table: 0x6000139d2ac0
   tick = 3244
   k = PERIODIC 116399: 0.100000
   v = true
   already_dead = nil
scripts/scheduler.lua:409 in (global) RunScheduler (Lua) <407-415>
   tick = 3244
scripts/update.lua:240 in () ? (Lua) <224-298>
   dt = 0.033333335071802
   tick = 3244
   i = 3244]
```

Seems we need check if `state.hook` exists

**Comment by penguin0616** [comment]
This was fixed in e1e22c8d4e97450981b76f6077af0b677478cca4, I've just been waiting to push it out until the entire update is complete.

**Comment by penguin0616** [comment]
This fix is now live on steam.
